auth-web/src/components/auth/common/AccountBusinessTypePicker.vue

```diff
@@ -0,0 +1,164 @@
+                    <v-col cols="6" >
+                        <v-select
+                        filled
+                        label="Business Type"
+                        item-text="desc"
+                        item-value="code"
+                        :items="businessTypeCodes"
+                        v-model="businessType"
+                        data-test="select-business-type"
+                        :rules="orgBusinessTypeRules"
+                        :menu-props="{ auto:true, offsetY: true, maxHeight: 400 }"
+                        ref="businessType"
+                        />
+                    </v-col>
+                    <v-col cols="6">
+                        <v-select
+                        filled
+                        label="Business Size"
+                        item-text="desc"
+                        item-value="code"
+                        :items="businessSizeCodes"
+                        v-model="businessSize"
+                        data-test="select-business-size"
+                        :rules="orgBusinessSizeRules"
+                        :menu-props="{ auto:true, offsetY: true, maxHeight: 400 }"
+                        ref="businessSize"
+                        />
+                    </v-col>
+                </v-row>
+              </v-expand-transition>
+            </template>
+        </v-form>
+      </div>
+    </v-container>
+</template>
+
+<script lang="ts">
+
+import { Component, Emit, Mixins, Prop, Vue, Watch } from 'vue-property-decorator'
+import { OrgBusinessType, Organization } from '@/models/Organization'
+import { Account } from '@/util/constants'
+import AccountChangeMixin from '@/components/auth/mixins/AccountChangeMixin.vue'
+import { Code } from '@/models/Code'
+import OrgNameAutoComplete from '@/views/auth/OrgNameAutoComplete.vue'
+import { namespace } from 'vuex-class'
+
+const OrgModule = namespace('org')
+const CodesModule = namespace('codes')
+
+@Component({
+  components: {
+    OrgNameAutoComplete
+  }
+})
+export default class AccountBusinessTypePicker extends Mixins(AccountChangeMixin) {
+  @Prop({ default: null }) errorMessage: string
+  @Prop({ default: false }) saving: boolean
+
+  @OrgModule.State('currentOrganization') public currentOrganization!: Organization
+
+  @CodesModule.Action('getBusinessSizeCodes') private readonly getBusinessSizeCodes!: () => Promise<Code[]>
+  @CodesModule.Action('getBusinessTypeCodes') private readonly getBusinessTypeCodes!: () => Promise<Code[]>
+  @CodesModule.State('businessSizeCodes') private readonly businessSizeCodes!: Code[]
+  @CodesModule.State('businessTypeCodes') private readonly businessTypeCodes!: Code[]
+
+  private isLoading = false
+
+  $refs: {
+    businessType: HTMLFormElement,
+    businessSize: HTMLFormElement
+  }
+
+  // input fields
+  private isBusinessAccount = false
+  private name = ''
+  private businessType = ''
+  private businessSize = ''
+  private branchName = ''
```

Comment:
this component contain only businessType and businessSize. are we using name and branchName